src/callbacks_step/save_restart_dg.jl

```diff
@@ -327,4 +327,53 @@ function load_restart_file_on_root(mesh::Union{ParallelTreeMesh, ParallelP4estMe
 
     return u_ode
 end
+
+function save_restart_controller(mesh, integrator,
```

Comment:
Why pass the `mesh` as an argument? It seems unused...

Reply:
You are right